airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -86,10 +86,9 @@ object TypeResolver extends LogSupport {
           case k @ GroupingKey(LongLiteral(i, _), _) if i <= selectItems.length =>
             // Use a simpler form of attributes
             val keyItem = selectItems(i.toInt - 1) match {
-              case SingleColumn(expr, _, _) =>
-                expr
-              case other =>
-                other
+              case a: Attribute =>
+                toResolvedAttribute(a.name, a)
+              case other => other
```

Comment:
Applied the same way as `resolveSortItemIndexes`:
https://github.com/wvlet/airframe/blob/322dc470ec941e881a674a95e98e52e18094a669/airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala#L136-L140